plugins/cloudtrail/rules/aws_cloudtrail_rules.yaml

```
@@ -21,7 +21,7 @@
 
 - required_plugin_versions:
   - name: cloudtrail
-    version: 0.10.0
+    version: 0.11.0
```

```suggestion
    version: 0.10.0
```
I don't think this needs to be bumped, there is no new feature used in the ruleset.